### PR TITLE
feat: Notification Center

### PR DESCRIPTION
**Pull request by XinyuCRO**
Closes https://github.com/crypto-com/chain-desktop-wallet/issues/1145

## Screenshot

image

image

## Notification Management

We use GitHub Repo to manage public notification. `config/notifications/dev.json` for testing and `config/notifications/release.json` for releasing notifications.

- dev endpoint: https://raw.githubusercontent.com/crypto-com/chain-desktop-wallet/dev/config/notifications/dev.json
- release endpoint: https://raw.githubusercontent.com/crypto-com/chain-desktop-wallet/dev/config/notifications/release.json

These two links point to the latest version of the notifications on the `dev` branch, which mean if we make changes to `config/notifications/release.json` and push it to the `dev` branch, we are pushing pubic notifications to users

Every time the client start and every 5 minutes, the client will fetch the endpoint, and determine whether there are new notifications, there is some cache-control on the endpoints, so there might be some slight delay

The detailed design can be seen here: https://github.com/crypto-com/chain-desktop-wallet/issues/1196

### CODE REVIEW

File: src/pages/notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                      size={48}
+                      icon={IconFromNotification(item)}
+                      style={{ backgroundColor: '#7B849B' }}
+                    />
+                  </div>
+                </div>
+                <div style={{ marginLeft: '10px', overflow: 'hidden' }}>
+                  <Typography.Paragraph
+                    style={{ fontSize: '14px' }}
+                    ellipsis={{ rows: 1, expandable: true }}
```
Comment (crypto-matto) [inline src/pages/notification/index.tsx:103]: May use `symbol` to input expand translation
```suggestion
                    ellipsis={{ rows: 1, expandable: true, symbol: t('general.notification.expand') }}
```

**Comment by crypto-matto** [comment]
Is there any expire mechanism for out-dated CS notifications? 
螢幕截圖 2022-08-02 下午2 19 17
Also I noticed that the imported notification is based on the language I selected at the time. 
I'm just thinking if we should instead store the index of translation here, for standardized notification
```
    postLocalNotification({
      content: 'general.customerService.notification.body',
      type: 'customerService'
    })
```
Then decode here.
``` 
<Typography.Paragraph
      style={{ fontSize: '14px' }}
      ellipsis={{ rows: 1, expandable: true }}
>
    {t(item.content)}
</Typography.Paragraph>
```
But I also realize that this use case is not common, as not much people will switch languages in between. 
You may decide whether to make the adjustment or not.

**Comment by XinyuCRO** [comment]
@crypto-matto 

>Is there any expire mechanism for out-dated CS notifications?

No

>Also I noticed that the imported notification is based on the language I selected at the time.

Yeah, not many cases, also users will get these notifications when they are using it, which means they can definitely read these different languages

**Comment by crypto-matto** [comment]
> @crypto-matto
> 
> > Is there any expire mechanism for out-dated CS notifications?
> 
> No

May be in a long run, we may apply a removal of notifications, during the notification list fetching. When we got `isRead`: true & `createdAt` was like 7 / 14 or 30 days ago, then we remove the notification from the list. But guess it's too early for this & the list won't be too long for most people, as not many would use Live Chat too often. 

Let's merge this then :+1:

**Review by crypto-matto — DISMISSED** [review]

**Pull request merged**